bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/TargetEnvironment.java

```diff
@@ -89,10 +91,25 @@ public static TargetEnvironment create(SPTarget base, GuideEnvironment guide, Im
     }
 
     /** Returns an arbitrary target from the asterism. This method will be removed for 18A. */
+    public SPSkyObject getPrimaryTargetFromAsterism() {
```

Comment:
The comment needs to be reworded, I think.  It isn't clear to me what the "primary" target means though.  Is it supposed to be the slew position?  Also, it can be an `SPCoordinate` so it isn't technically necessarily a target in the `Target` ADT sense?

Reply:
Agreed. I'm going to release a PR next getting rid of the `getArbitraryTargetFromAsterism` next. In the single target case, it was the only science target, whereas now I believe we should be using it as the slew position. I'll change this here now, though, to `getSlewPositionObjectFromAsterism`?

Reply:
Seems good.